src/indexer/compose.yaml

```
@@ -0,0 +1,75 @@
+---
+services:
+  broker:
+    build:
+      context: '.'
+      dockerfile: 'broker/Dockerfile'
+    depends_on:
+      processor:
+        condition: 'service_started'
+    environment:
+      PROCESSOR_WS_URL: 'ws://processor:3008/ws'
+      RUST_LOG: 'info'
+    image: 'econialabs/emojicoin-broker'
+    ports:
+    - '3009:3009'
```

Env var in case AWS has issues

---

For port 3008/3009

---

```suggestion
```suggestion
    - '${PROCESSOR_WS_PORT}:${PROCESSOR_WS_PORT}'
```
```

---

Addressing in ECO-2115